.github/workflows/ci-pipeline.yaml

```diff
@@ -40,14 +40,15 @@ jobs:
     name: Azure Container Service Publish
     runs-on: ubuntu-latest
     environment: Build and Test environment
+    needs: docker
     steps:
       - name: Azure Container Service Publish
         #uses: actions/checkout@v2
         run: curl -dH -X POST ${{ secrets.AZURE_WEBHOOK_URL }}
 
   docker:
     needs: build
-    if: ${{github.ref_name == 'master'}}
+    if: ${{github.ref_name == 'module10'}}
```

Comment:
Thanks for checking this bit still plays ball - don't forget to switch it back when you're ready to merge 🙂